blockstack_client/snv.py

```diff
@@ -932,22 +1004,25 @@ def snv_lookup(verify_name, verify_block_id, trusted_serial_number_or_txid_or_co
         # but that's okay--if the consensus hash in this tx is inauthentic, it will be unreachable
         # from the other consensus hash [short of a SHA256 collision])
         trusted_block_id = get_block_from_consensus(trusted_consensus_hash, proxy=proxy)
-        if type(trusted_block_id) == dict and 'error' in trusted_block_id:
+        if isinstance(trusted_block_id, dict) and 'error' in trusted_block_id:
             # got error back
             return trusted_block_id
-
     else:
-        return {'error': 'Did not receive a valid txid, consensus hash, or serial number (%s)' % trusted_serial_number_or_txid_or_consensus_hash}
+        msg = 'Did not receive a valid txid, consensus hash, or serial number ({})'
+        return {'error': msg.format(trusted_serial_number_or_txid_or_consensus_hash)}
 
     if trusted_block_id < verify_block_id:
-        return {'error': 'Trusted block/consensus hash came before the untrusted block/consensus hash'}
+        msg = 'Trusted block/consensus hash came before the untrusted block/consensus hash'
+        return {'error': msg}
 
     # go verify the name
     verify_consensus_hash = get_consensus_at(verify_block_id, proxy=proxy)
-    historic_namerec = snv_name_verify(verify_name, trusted_block_id, trusted_consensus_hash, verify_block_id, verify_consensus_hash)
+    historic_namerec = snv_name_verify(
+        verify_name, trusted_block_id, trusted_consensus_hash,
+        verify_block_id, verify_consensus_hash
+    )
 
     return historic_namerec
 
 # backwards compatibility
 lookup_snv = snv_lookup
-
```

Comment:
Thank you for taking a crack at this file!  I love that you left comments in here about whether or not variables hadn't been declared, or whether or not variables were unused :)